task3/simplyTable.js

```diff
@@ -0,0 +1,18 @@
+let mybtn = document.getElementById('btn')
```

Comment:
In common everything works good, but not obviously solution. Lets change algorithm to something more readable:
```
setTimeout(function () {
        const interval = setInterval(function () {
        console.log('Check started')
        const pendingStatuses = document.querySelectorAll('.pending')
        const countOfElementsToChange = pendingStatuses.length
        if (countOfElementsToChange) {
          pendingStatuses[0].classList.remove('pending')
          pendingStatuses[0].innerHTML = 'Done'
          pendingStatuses[0].style.color = '#85929E'
        } else {
          clearInterval(interval)
        }
```